.zenodo.json

```
@@ -33,6 +36,11 @@
             "name": "Burrell, A.G.",
             "orcid": "0000-0001-8875-9326"
         },
+        {
+            "affiliation": "Virginia Polytechnic Institute and State University",
```

I believe "Virginia Tech" is their official name now (https://en.wikipedia.org/wiki/Virginia_Tech)
```suggestion
            "affiliation": "Virginia Tech",
```

---

Our official name is still the same, the "Tech" part is more of a shorthand/jargon since the full name doesn't roll off the tongue very well when reported in the news.  Either of these seems to be accepted by ORCID too.  It's really up to the author I think, so that would be more a question for @SEC-VT.

---

Sorry for jumping in here where I haven't done much with pydarn development.  Did I miss any communications from @SEC-VT as I see this was changed with 39ef7d4?  I think it might be best to consult with him as either version is acceptable and it's a personal preference.  But forgive me if I've missed a note or comment here or if there's another thought for the change that I'm not understanding.